aip/general/0009.md

```diff
@@ -122,12 +138,18 @@ Examples of complexities that IaC clients abstract away include:
 
 [Terraform][] is an example of such a client.
 
+### User
+
+A human being which is using an API. This term is defined to differentiate usage
```

Comment:
How does a human use an API? I know how to walk and smell, but I don't have a "go go gadget call API" in my repertoire. I would say "directly" and give an example like, "with cURL".

Reply:
sounds good, added.